types/react/index.d.ts

```diff
@@ -2593,6 +2593,7 @@ declare namespace React {
         summary: DetailedHTMLFactory<HTMLAttributes<HTMLElement>, HTMLElement>;
         sup: DetailedHTMLFactory<HTMLAttributes<HTMLElement>, HTMLElement>;
         table: DetailedHTMLFactory<TableHTMLAttributes<HTMLTableElement>, HTMLTableElement>;
+        template: DetailedHTMLFactory<HTMLAttributes<HTMLElement>, HTMLElement>;
```

Comment:
Looks like [`templaate`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template) even implements [its own interface](https://developer.mozilla.org/en-US/docs/Web/API/HTMLTemplateElement). Is this available in the dom lib?

Reply:
> Looks like [`templaate`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template) even implements [its own interface](https://developer.mozilla.org/en-US/docs/Web/API/HTMLTemplateElement). Is this available in the dom lib?

Yes. It's my fault to forget this. I'll correct it.